src/StatusQ/Popups/StatusSearchPopup.qml

```diff
@@ -175,8 +175,9 @@ StatusModal {
                                 spacing: 2
                                 StatusBaseText {
                                     color: Theme.palette.directColor1
-                                    text: searchOptionsMenuButton.prefixText + ": "
```

Comment:
not sure if this `prefixText` is used somewhere?

Reply:
@alexandraB99 it's not, I checked :)